### PR TITLE
finished seeding script. Generated and loaded the fake data into the …

### PR DESCRIPTION
**Pull request by wwong007**
…database. Set up server to handle get routes from different endpoints.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const port = 3000;
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.listen(port, () => console.log('APP IS LISTENING'));
+
+app.use('/', express.static(path.join(__dirname, './client/dist')));
+
+app.get('/api/stocks/:symbol', (req, res) => {
```
Comment (jaytxng) [inline server/index.js:17]: reminder to refactor endpoint to '/stocks/:symbol' structure down the line

**Review by jaytxng — APPROVED** [review]

File: database/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const mongoose = require('mongoose');
+
 const mongoUri = 'mongodb://localhost/infoModule';
 
 const db = mongoose.connect(mongoUri);
```
Comment (jordanstholmes) [inline database/index.js:5]: @wwong007 @iamjaytong Probably a good idea to handle errors that might come from connecting to the database. Might also want some kind of console log indicating a successful connection has been made (if indeed it has)

File: database/seed.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    yearLow: (parseInt(todayHigh) - 28.03),
+    tags: collectionsArr,
+  };
+  const insertSampleStocks = () => {
+    Stock.create(sampleStocks)
+      .then(() => db.close())
+      .catch(err => console.log('errrerererer', err));
+  };
+  insertSampleStocks();
+}
```
Comment (jordanstholmes) [inline database/seed.js:79]: @wwong007 Very well done seeding script :)

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+const path = require('path');
+const Stock = require('../database/Stock.js');
+
+const app = express();
+const port = 3000;
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.listen(port, () => console.log('APP IS LISTENING'));
```
Comment (jordanstholmes) [inline server/index.js:13]: @wwong007 @iamjaytong Consider logging what port it's listening on. Comes in handy.

File: server/index.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+app.use('/', express.static(path.join(__dirname, './client/dist')));
+
+app.get('/api/stocks/:symbol', (req, res) => {
+  Stock.find(req.params, (err, data) => {
+    console.log(req.params);
```
Comment (jordanstholmes) [inline server/index.js:19]: @wwong007 @iamjaytong Try not to leave stray console.logs that will be merged into master

File: server/index.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    if (err) {
+      throw err;
```
Comment (jordanstholmes) [inline server/index.js:21]: @wwong007 @iamjaytong throwing an error here will crash the server. That is not something we want. Even if there was an error, we still want to send a response to the client with an appropriate status code.

**Pull request merged**